lib/angulartics-newrelic-insights.js

```diff
@@ -16,6 +16,7 @@
     .config(['$analyticsProvider', function($analyticsProvider) {
       angulartics.waitForVendorApi('newrelic', 100, function(newrelic) {
         $analyticsProvider.registerEventTrack(function(action, properties) {
+          properties.eventType = 'PageAction';
```

Comment:
I don't disagree with you at all. Here is the issue. I have a element <a class="btn btn-sm btn-warning"
                                   analytics-on
                                   analytics-event="itemAddedToCart-productView"
                                   analytics-category="cart"
                                   analytics-label={{productCtrl.product.id}}
                                   ng-click="productCtrl.adjustCartQuantity(productCtrl.product)"
                                   ng-if="productCtrl.showAddToCartButton()">

this sends an eventType of "click" to new relic. New Relic responses with a 400 Bad Request. Do you have another suggestion for me to look at? or would it be better for me to change a few values ie if eventType === 'click' then eventType = 'PageAction' so we are deliberate in how the object is modified?